ast/infix_expression.go

```diff
@@ -28,6 +28,9 @@ func (oe *InfixExpression) String() string {
 		out.WriteString(oe.Left.String())
 	}
 	out.WriteString(" " + oe.Operator + " ")
+	if oe.Right == nil {
+		panic("Missing \"%>\" closing tag")
```

Comment:
Not a fan of this `panic` here. Especially since the idea is to "fix a panic". Any other ways we can fix that?

Reply:
The idea here was to give some usable info to fix the real problem. Maybe we can find a better way.